board/common/overlay/sbin/fwupdate

```diff
@@ -112,6 +112,26 @@ function do_download() {
     rm -f $FW_DIR/$FW_FILE_GZ $FW_DIR/$FW_FILE_XZ
```

Comment:
If we `rm -rf $FW_DIR/*` three lines below, we could remove these individual `rm` commands.

Reply:
Very good catch. I have removed the redundant rm commands.